packages/cactus-plugin-something-somehow/src/main/typescript/implementation/jest.config.js

```diff
@@ -1,7 +1,7 @@
 /*
```

Comment:
Would you keep the location of /packages/jest.config.js? (not move this file to another directory)

Reply:
@takeutak Done. Put it back to `./packages/`